fuzzinator/listener/listener_manager.py

```diff
@@ -12,6 +12,19 @@
 
 logger = logging.getLogger(__name__)
 
+class Trampoline(object):
```

Comment:
Isn't two empty lines before/between top-level entities the style?

Reply:
Ohh yeah, true. Will fix it.